layers/+tools/lsp/funcs.el

```diff
@@ -125,7 +124,11 @@
     "x" "text/code"
     "xh" #'lsp-document-highlight
     "xl" #'lsp-lens-show
-    "xL" #'lsp-lens-hide))
+    "xL" #'lsp-lens-hide)
+  (when (configuration-layer/package-used-p 'treemacs)
```

Comment:
the test should be on `lsp-treemacs`. it requires `treemacs`. so essentially both packages are checked here.